x-pack/plugins/observability/public/pages/alerts/status_filter.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiFilterButton, EuiFilterGroup } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+import type { AlertStatus } from '../../../common/typings';
+
+export interface StatusFilterProps {
+  status: AlertStatus;
+  onChange: (value: AlertStatus) => void;
+}
+
+export function StatusFilter({ status = 'open', onChange }: StatusFilterProps) {
+  return (
+    <EuiFilterGroup>
+      <EuiFilterButton
```

Comment:
I think we might want to add some aria description so it says something like "Click to filter for recovered alerts".

Reply:
Made it "Filter alerts by active and recovered status" and added in 43986303304.